src/components/video-editor/timeline/TimelineEditor.tsx

```
@@ -670,16 +687,16 @@ export default function TimelineEditor({
   onAudioDelete,
   selectedAudioId,
   onSelectAudio,
-  aspectRatio,
-  onAspectRatioChange,
+  aspectRatio = 'native',
+  onAspectRatioChange = () => {},
```

Biome (2.4.11)

[error] Unexpected empty block.

(lint/suspicious/noEmptyBlockStatements)